utils/python/CIME/XML/env_batch.py

```diff
@@ -330,6 +330,8 @@ def get_submit_args(self, case, job):
             if name is None:
                 submitargs+=" %s"%flag
             else:
+                if name.startswith("$"):
```

Comment:
In config_batch.xml the submit_args node references other xml variables, we need to strip the $ from the front of the variable so that we can resolve it. We have to do this explicitly because this is an attribute not a node,